src/Gui/ToolBarManager.cpp

```diff
@@ -202,6 +204,9 @@ void ToolBarManager::setup(ToolBarItem* toolBarItems)
                 QApplication::translate("Workbench",
                                         toolbarName.c_str())); // i18n
             toolbar->setObjectName(name);
+            if (nameAsToolTip){
+                toolbar->setToolTip(QLatin1String("[") + name + QLatin1String("]"));

```

Comment:
The 'name' is hereby the technical/internal toolbar name. Sine you expose it as tooltip, it must be translatable.

This line instead fixes the issue for me:

`toolbar->setToolTip(QApplication::translate("Workbench", toolbarName.c_str()));`

I would omit the brackets in the tooltip but this is of course a matter of taste.

Reply:
Okay, I'll make it translated.  I wanted the brackets to distinguish it from other tooltips for the various commands.